tests/test_ase_setup/otf_setup.py

```diff
@@ -12,7 +12,7 @@
 
 np.random.seed(12345)
 
-md_engines = ['VelocityVerlet', 'NVTBerendsen', 'NPTBerendsen', 'NPT']
+md_engines = ['VelocityVerlet'] #, 'NVTBerendsen', 'NPTBerendsen', 'NPT']
```

Comment:
Is NVT available?

Reply:
It is available. `otf_setup.py` is not the unit test file, the unit test file is `test_otf.py` which contains all MD engines. I modified it only because I used it in the documentation (tutorial)